README.md

```diff
@@ -433,36 +434,16 @@ def self.call(input)
 end
 ```
 
+## Contributing to Sprockets
 
-## Development
-
-### Contributing
-
-The Sprockets source code is [hosted on
-GitHub](https://github.com/rails/sprockets). You can check out a
-copy of the latest code using Git:
-
-    $ git clone https://github.com/rails/sprockets
+Sprockets is the work of hundreds of contributors. You're encouraged to submit pull requests, propose
+features and discuss issues.
 
-If you've found a bug or have a question, please open an issue on the
-[Sprockets issue
-tracker](https://github.com/rails/sprockets/issues). Or, clone
-the Sprockets repository, write a failing test case, fix the bug and
-submit a pull request.
+See [CONTRIBUTING](CONTRIBUTING.md).
 
 ### Version History
 
 Please see the [CHANGELOG](https://github.com/rails/sprockets/tree/master/CHANGELOG.md)
 
-## Build Status
-
-[![Build Status](https://travis-ci.org/rails/sprockets.svg?branch=master)](https://travis-ci.org/rails/sprockets)
```

Comment:
Can you move the travis badge to the `##  contributing to sprockets` section and I think this is good to merge. Thanks for your work :heart: